WMFData/Sources/WMFData/Data Controllers/Donor Experience/WMFDonateLocalHistory.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+public final class WMFDonateLocalHistory: Codable {
+
+    public let donationTimestamp: String
+    public let donationType: DonationType
+    public let donationAmount: Decimal
```

Comment:
I think we should save the currencyCode here as well - it'll help us aggregate them and display properly in the future.